apps/daffio/src/app/core/header/containers/header.component.ts

```diff
@@ -8,17 +8,17 @@ import { ToggleSidebar } from '../../sidebar/actions/sidebar.actions';
   selector: 'daffio-header-container',
   templateUrl: './header.component.html'
 })
-export class DaffioHeaderContainer {
-  constructor(private store: Store<{}>) { }
-
-  openSidebar() {
-    this.store.dispatch(new ToggleSidebar());
-  }
-
+export class DaffioHeaderContainerComponent {
```

Comment:
I think this should be named `DaffioHeaderContainer`.

Reply:
resolved in dbcf3deb24b3190d5c086b952e576b6e7fca991d

Reply:
resolved in 5ac93b24b23fdc12d649d1253f6285edde617b6b